### DISCUSSION TITLE
No explanation for full rooms - no spectate option

### DISCUSSION THREAD

**Discussion by wmurphyrd** [mozilla/hubs#4110] [General Questions & Discussion] [answered]
Perhaps my memory is failing me, but I thought the spectate option used to be allowed even when a room was full. With the current hubs (screenshot from hubs.mozilla.com today), opening a full room just shows an entry panel with no options and no explanation. Is this intended?

![image](https://user-images.githubusercontent.com/10034859/113420954-da7a8100-938f-11eb-8396-7b18408f1242.png)

**Comment by johnshaughnessy** [comment, marked as answer]
I believe this is a bug. I've converted it into an issue. https://github.com/mozilla/hubs/issues/4116
